erpnext/projects/doctype/project/project.py

```diff
@@ -2139,3 +2150,28 @@ def set_warranty_claim_denied(projects, denied, reason=None):
 		doc.warranty_claim_denied = denied
 		doc.warranty_claim_denied_reason = reason
 		doc.save()
+
+
+@frappe.whitelist()
+def submit_feedback(project, customer_feedback):
+	if not customer_feedback:
+		frappe.throw(_('Remarks are mandatory for Communication'))
+
+	if not frappe.db.exists('Project', project):
+		frappe.throw(_("Repair Order {0} does not exist".format(project)))
+
+	target_doc = frappe.get_doc('Project', project)
+
+	cur_dt = get_datetime()
+	cur_date = getdate(cur_dt)
+	cur_time = get_time(cur_dt)
+
+	target_doc.feedback_date = cur_date
+	target_doc.feedback_time = cur_time
+	target_doc.customer_feedback = customer_feedback
+
+	target_doc.save()
```

Comment:
How about lets not .save() and run `target_doc.validate_feedback()` then `target_doc.db_set({k: v, k: v}, None)`
To prevent permission issues. Service CRM may not have write permission for Project